src/sentry/utils/kvstore/bigtable.py

```diff
@@ -65,6 +65,7 @@ def __init__(
         client_options: Optional[Mapping[Any, Any]] = None,
         default_ttl: Optional[timedelta] = None,
         compression: Optional[str] = None,
+        app_profile: Optional[str] = None,
```

Comment:
This change could have been done separately but it seems minor enough to just bring in here.